expression/builtin_arithmetic.go

```diff
@@ -40,13 +41,19 @@ var (
 	_ builtinFunc = &builtinArithmeticMinusDecimalSig{}
 	_ builtinFunc = &builtinArithmeticMinusIntUnsignedSig{}
 	_ builtinFunc = &builtinArithmeticMinusIntSig{}
+	_ builtinFunc = &builtinArithmeticDivideRealSig{}
+	_ builtinFunc = &builtinArithmeticDivideDecimalSig{}
 	_ builtinFunc = &builtinArithmeticMultiplyRealSig{}
 	_ builtinFunc = &builtinArithmeticMultiplyDecimalSig{}
 	_ builtinFunc = &builtinArithmeticMultiplyIntUnsignedSig{}
 	_ builtinFunc = &builtinArithmeticMultiplyIntSig{}
 	_ builtinFunc = &builtinArithmeticSig{}
 )
 
+// precIncrement indicates the number of digits by which to increase the scale of the result of division operations
+// performed with the / operator.
+const precIncrement = 4
```

Comment:
if it's only used in type inference of `DIV`, it's better to be a member of `arithmeticDivideFunctionClass`